specs/backlog.md

```diff
@@ -0,0 +1,92 @@
+
+1. Issues should be written associated with the repository closest to the section of relevant code.  For example a hil/bmi issues would be written in CCI-MOC/hil. If no repository is appropriate they may be added directly to the [mocbacklog](https://github.com/orgs/CCI-MOC/projects/8).   
+2. If you add the issue to the repository,whenever possible add the repository to [Linked Repositories](https://github.com/orgs/CCI-MOC/projects/8/settings/linked_repositories) under the Menu.  This way it will be included in default searches.  
+3. To find and order all cards available across all cci-moc repositories execute this search in the [add card](assets/githubaddcards.png) dialog:
+ 
+    `org:CCI-MOC state:open is:issue is:open repo:*`
+ 
+### Backlog Setup/Additions
+1. Open [mocbacklog](https://github.com/orgs/CCI-MOC/projects/8)
+2. Click + Add cards on the right side of the project board
+3. In the search field use this search: `org:CCI-MOC is:open is:issue repo:*`,  making sure "Only show results from linked repositories" is *not* checked.
+4. Drag the cards that show in the search results to the BackLog column add them <--- as a team figure out if we need two columns, one for order or "Ready to be worked" --->.
+   It should look something like [this](assets/fillbacklog.png) when you do it. 
+    
+### Scrum Setup
+1. Open [mocbacklog](https://github.com/orgs/CCI-MOC/projects/8) 
+2. Create a new Scrum Project with the name based on starting and end dates - eg. Sprint week 46 and 47 based on number for weeks of year.  
+3. In Sprint Planning meeting move items from Backlog to Sprint Project. 
+
+This is part of a larger effort to make the work of managing the MOC, and by extension the [Open Infra Labs](https://openinfralabs.org) work open and more inviting for partners and the community to participate in.  This is not intended to capture all the work being done on the MOC, rather it is focused on the core dev and ops teams work.   If this changes over time we will revisit the policy and make adjustments. 
+
+## Alternatives & History
+
+Other approaches:
+1. Use Github Kanban model and do not have separate Scrum Boards for each sprint.   <--- **Honestly given our size we should probably shelve sprints and do Kannan.  What does team need to see to decide?** --->
+2. Automate the moving of the backlog between scrum boards every sprint. 
+
+## Implementation
+
+### Author(s)
+
+Primary author:
+  - Michael Daitzman <msd@bu.edu>
+
+Other contributors:
+  - team discussion
+
+### Milestones
+
+Policy will go into affect on Nov 16, 2020.
```

Comment:
This is awfully optimistic. Possibly "Policy will go into affect when this proposal is accepted by the team".

Reply:
Well, we're strating the sprint on the 16th, so I'll leave it for now :-)